examples/minimal/tamr-vm.tf

```diff
@@ -2,12 +2,12 @@ module "tamr-vm" {
   source = "git::git@github.com:Datatamer/terraform-emr-tamr-vm?ref=4.1.0"
 
   ami                = var.ami_id
-  instance_type      = "m4.2xlarge"
+  instance_type      = "r5.2xlarge"
   key_name           = module.emr_key_pair.key_pair_key_name
   subnet_id          = var.ec2_subnet_id
   vpc_id             = var.vpc_id
   security_group_ids = module.aws-sg-vm.security_group_ids
-
+  availability_zone = data.aws_subnet.application_subnet.availability_zone
```

Comment:
Formatting seems off here, you may need to run `terraform fmt`

Reply:
Fixed [here](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/20f40c013e2fa8cc62d9a987df2deb829e6cd98d). Also [added a variable](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/1f9516c4214df62726578e886e889dc16526ce5b) that was missing